package.json

```diff
@@ -2,9 +2,12 @@
   "name": "test_track_js_client",
   "version": "1.3.8",
   "description": "Javascript Client for Test Track",
-  "main": "dist/testTrack.js",
+  "main": "dist/testTrack.bundle.js",
+  "module": "dist/testTrack.js",
```

Comment:
This is a new standard for exporting modules from npm packages: https://github.com/rollup/rollup/wiki/pkg.module